operator/src/main/java/oracle/kubernetes/operator/helpers/EventHelper.java

```diff
@@ -615,6 +622,42 @@ public V1ObjectReference createInvolvedObject(EventData eventData) {
       protected String generateEventName(EventData eventData) {
         return generateOperatorNSEventName(eventData);
       }
+    },
+    CONVERSION_WEBHOOK_FAILED {
+      @Override
+      protected String getType() {
+        return EVENT_WARNING;
+      }
+
+      @Override
+      public String getReason() {
+        return CONVERSION_WEBHOOK_FAILED_EVENT;
+      }
+
+      @Override
+      public String getPattern() {
+        return DOMAIN_CONVERSION_FAILED_PATTERN;
```

Comment:
Okay, the constant with the English text is used here and is used in a user visible string... This needs to come from the resource bundle.

From DomainValidationMessages.java (you can do this):
```
  private static String getBundleString(String key) {
    return ResourceBundle.getBundle("Operator").getString(key);
  }
```

Reply:
Changed to get it from resource bundle and use LOGGER.formatMessage().